Specifications/Tool/com.apple.commands.built-in.validate.xcspec

```diff
@@ -0,0 +1,26 @@
+/**
+ Copyright (c) 2015-present, Facebook, Inc.
+ All rights reserved.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree. An additional grant
+ of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+{
+    Type = Tool;
+    Identifier = com.apple.build-tools.platform.validate;
+    Name = Validate;
+
+    CommandLine = "builtin-validationUtility [options] [input]";
+
+    Outputs = (
+        "$(TEMP_FILE_DIR)/com.apple.build-tools.platform.validate"
+    );
+
```

Comment:
Missing the environment variables:

    EnvironmentVariables = {
        "PRODUCT_TYPE" = "$(PRODUCT_TYPE)";
    };